test_helpers/pool_helper.go

```diff
@@ -0,0 +1,239 @@
+package test_helpers
+
+import (
+	"fmt"
+	"reflect"
+	"time"
+
+	"github.com/tarantool/go-tarantool"
+	"github.com/tarantool/go-tarantool/connection_pool"
```

Comment:
It means that ConnectionPool will be imported even for non-Pool tests. I don't know how bad it is (highly likely no one will notice), but it may cause problems